tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -1785,7 +1793,9 @@ void run_tests() {
     test_removal<Alloc>();
     test_misc<Alloc>();
     test_sstream<Alloc>();
+#if !defined(__clang__) && !defined(_M_IX86) // TRANSITION, LLVM-54804
     test_exceptions<Alloc>();
+#endif // !defined(__clang__) && !defined(_M_IX86)

```

Comment:
This guard is incorrect. LLVM-54804 is specific to Clang x86, so we want to run this code for !(Clang && x86), which would De Morgan to !Clang || !x86, but is probably simpler to understand without De Morganing.